gql/dsl.py

```diff
@@ -84,7 +84,7 @@ def select(self, *fields):
         added_selections = selections(*fields)
         if selection_set:
             selection_set.selections = FrozenList(
-                selection_set.selections + added_selections
+                selection_set.selections + [selection for selection in added_selections]
```

Comment:
Good catch!

A list comprehension is not necessary, though. You can do
```py
                selection_set.selections + list(added_selections)
```
or maybe
```py
                chain(selection_set.selections, added_selections)
```
with
```py
from itertools import chain
```

Reply:
You are right. I went with your first solution as it is more readable.